client/landing/gutenboarding/onboarding-block/style-preview/preview.tsx

```diff
@@ -11,13 +12,110 @@ import { STORE_KEY } from '../../stores/onboard';
 import * as T from './types';
 import { ValuesType } from 'utility-types';
 
+type Design = import('../../stores/onboard/types').Design;
+type SiteVertical = import('../../stores/onboard/types').SiteVertical;
+
 interface Props {
 	viewport: T.Viewport;
 	fonts: ValuesType< typeof import('../../constants').fontPairings >;
 }
 const Preview: React.FunctionComponent< Props > = ( { fonts, viewport } ) => {
-	const { selectedDesign } = useSelect( select => select( STORE_KEY ).getState() );
-	const [ { title: fontA }, { title: fontB } ] = fonts;
+	const [ previewHtml, setPreviewHtml ] = React.useState< string >();
+	const [ requestedFonts, setRequestedFonts ] = React.useState< Set< string > >( new Set() );
+
+	const { selectedDesign, siteVertical } = useSelect( select =>
+		select( STORE_KEY ).getState()
+	) as { selectedDesign: Design; siteVertical: SiteVertical };
```

Comment:
Why do we need the cast? :thinking:

Reply:
I had a comment at some point, seems a good practice to accompany cases with comments. I'll add another.

The type is possibly undefined, but our flow should guarantee that on this step we have both of these defined.